options.js

```diff
@@ -36,6 +36,7 @@ class Options {
         this.submitServerAssignmentName = submitServerAssignmentName;
         this.filesToCheck = filesToCheck;
         this.lateScoreAdjustment = -12;
+        this.students = "*";
```

Comment:
it occurs to me that one could take advantage of dynamic typing in javascript and allow either a string ("*") or an Array (["list","of","student","ids"]), but that may possibly be uglier than the current solution of allowing a string to specify multiple different things depending on formatting.

it could also be viable to disallow such flexibility, and simply specify a range of students using a firstStudent and lastStudent field (or similar). Non-contiguous distributions would simply be disallowed and if one wanted to be able to have the REVIEW links on all students, just specify "a" and "z" as the bounds

Reply:
the only reason i didn't do this to begin with was because i wanted to be able to copy paste the list of names provided in grading distributions, but it would be just as easy to modify the grading distribution code to only include the first and last student or just have the TA manually copy/paste the first and last student.

Reply:
Flexibility reduced; bounds specified as "a" and "}" for all students (need ASCII char after "z" to include directory IDs starting with "z"